src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -354,86 +404,101 @@ def execution_environment_volume_mounts(
         entry: SettingsEntry,
         config: ApplicationConfiguration,
     ) -> PostProcessorReturn:
+        """Post process set_environment_variable.
+
+        :param entry: The current settings entry
+        :param config: The full application configuration
+        :return: An instance of the standard post process return object
+        """
         # pylint: disable=unused-argument
         # pylint: disable=too-many-branches
-        """Post process set_environment_variable"""
+        # pylint: disable=too-many-locals
+
         messages: List[LogMessage] = []
         exit_messages: List[ExitMessage] = []
-        if entry.value.source in [
-            C.ENVIRONMENT_VARIABLE,
-            C.USER_CLI,
-        ]:
-            volume_mounts = flatten_list(entry.value.current)
-            for mount_path in volume_mounts:
-                parts = mount_path.split(":")
-                if len(parts) > 3:
+        entry_name = entry.settings_file_path(prefix="")
+        entry_source = entry.value.source
+        volume_mounts: List[VolumeMount] = []
+
+        if entry_source in (C.ENVIRONMENT_VARIABLE, C.USER_CLI):
+            hint = (
+                "Try again with format <source-path>:<destination-path>:<labels>'."
+                " Note: label is optional."
+            )
+            mount_strings = flatten_list(entry.value.current)
+            for mount_str in mount_strings:
+                src, dest, labels, *left_overs = chain(mount_str.split(":"), repeat("", 3))
```

Comment:
```suggestion
                src, dest, options, *left_overs = chain(mount_str.split(":"), repeat("", 3))
```

Reply:
Nah, I think we need to make the switch all in one PR

Reply:
I can do another PR to switch it everywhere all at once, but right now I think anything facing the user should be label because that is how it is documented...